dev-scripts/mark_rita_db_finished.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Usage: mark_rita_db_finished.sh DB_Name [mongo parameters]
+
+# This script is helpful if you have stopped ipfix-rita and would
+# like to analyze the results it has collected so far
+
+require_util () {
+	#Returns true if all binaries listed as parameters exist somewhere in the path, False if one or more missing.
```

Comment:
Could we pull this comment in line with the code. Non-critical so feel free to ignore